<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cam/webcam.png</normaloff>:/cam/webcam.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>100</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0.142045 rgba(134, 0, 231, 255), stop:0.522727 rgba(207, 41, 222, 255), stop:0.846591 rgba(248, 93, 145, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>730</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:0px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_correction">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/3d-printer.png</normaloff>Icon/3d-printer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Plus">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/plus.png</normaloff>Icon/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Minus">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/minus (1).png</normaloff>Icon/minus (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="Sensitivity_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 7pt &quot;맑은 고딕&quot;;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>보통</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Logout">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/logout (1).png</normaloff>Icon/logout (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_run">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:30px;
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/play-button.png</normaloff>Icon/play-button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:30px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/stop.png</normaloff>Icon/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Community">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:30px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/renewable-energy.png</normaloff>Icon/renewable-energy.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CamOn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:30px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/webcam.png</normaloff>Icon/webcam.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Logout_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 7pt &quot;맑은 고딕&quot;;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string>로그아웃</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Community_label">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 7pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>커뮤니티</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Measurement_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 6pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>흐트러짐 측정</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Start_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 7pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>실행</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Stop_label">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 7pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>정지</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Measurement">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/measuring.png</normaloff>Icon/measuring.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="Correction_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 6pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>정자세 보정</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Recode">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Icon/premium-icon-video-record-4348090.png</normaloff>Icon/premium-icon-video-record-4348090.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>43</width>
        <height>43</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="Measurement_label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 6pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="text">
       <string>기록</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>Sensitivity_label</zorder>
     <zorder>pushButton_CamOn</zorder>
     <zorder>pushButton_Plus</zorder>
     <zorder>pushButton_Minus</zorder>
     <zorder>pushButton_stop</zorder>
     <zorder>pushButton_correction</zorder>
     <zorder>pushButton_Logout</zorder>
     <zorder>pushButton_run</zorder>
     <zorder>pushButton_Community</zorder>
     <zorder>Community_label</zorder>
     <zorder>Logout_label</zorder>
     <zorder>Measurement_label</zorder>
     <zorder>Start_label</zorder>
     <zorder>Stop_label</zorder>
     <zorder>pushButton_Measurement</zorder>
     <zorder>Correction_label</zorder>
     <zorder>pushButton_Recode</zorder>
     <zorder>Measurement_label_2</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
